examples/build_scripts/build_gaea.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

But more to the point.. Nothing here requires bash. Just use `sh` :)

Reply:
Why?

Reply:
I see it similar to passing more info to a function than it needs. You use the tool that fills the requirements, and do not rely on more than is needed.

That said, people are probably copy-pasting out of this script rather than running it outright, so it's probably fine here.

Reply:
The short answer is [environment and platform portability](https://stackoverflow.com/questions/16365130/what-is-the-difference-between-usr-bin-env-bash-and-usr-bin-bash).

Reply:
we don't need platform portability if we're just running on Gaea ;-)

Reply:
Yeah this doesn't really matter. Only in that if every bash script had `#!/usr/bin/env bash` it would always work and nobody would ever need to have this discussion or think about which one is appropriate for the situation.